### PR TITLE
Packages

### PR DESCRIPTION
**Pull request by gregburek**
Adds precise and trusty packages via travis and packagecloud.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,45 @@
...
   - 1.4
+env:
+  global:
+    - COMPOSE_VERSION=1.5.1
+before_install:
+  - curl -L https://github.com/docker/compose/releases/download/${COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+  - sudo apt-get update -qq
+  - sudo apt-get install -qq liblz4-dev
```
Comment (cyberdelia) [inline .travis.yml]: You don't need to apt-get update, nor to install liblz4.

File: .travis.yml [hunk 2]
```diff
@@ -1,10 +1,45 @@
...
 before_script:
   - go get golang.org/x/tools/cmd/cover
   - go get golang.org/x/tools/cmd/vet
-script: go vet ./... && go test -cover ./...
+script:
+  - go vet ./... && go test -cover ./...
```
Comment (cyberdelia) [inline .travis.yml:18]: Maybe add `-race` to `go test` ? Unless it trigger more problem.

File: docker/precise/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:precise
+
+ENV DEBIAN_FRONTEND noninteractive
+
+ENV GOLANG_VERSION 1.5.1
```
Comment (cyberdelia) [inline docker/precise/Dockerfile]: 1.5.2 is out, nothing urgent.

File: docker/trusty/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+ENV GOLANG_DOWNLOAD_URL https://golang.org/dl/go$GOLANG_VERSION.linux-amd64.tar.gz
+ENV GOLANG_DOWNLOAD_SHA1 46eecd290d8803887dec718c691cc243f2175fe0
+
+ENV GOPATH /usr/share/gocode
+ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
+
+RUN apt-get update -qq && apt-get dist-upgrade -qq -y \
+  && apt-get install -qq --no-install-recommends -y \
+    build-essential ubuntu-dev-tools debhelper devscripts curl \
+    libdistro-info-perl git liblz4-dev dh-golang
```
Comment (cyberdelia) [inline docker/trusty/Dockerfile]: Again no need for liblz4.

File: docker/precise/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+RUN curl -fsSL "$GOLANG_DOWNLOAD_URL" -o golang.tar.gz \
+	&& echo "$GOLANG_DOWNLOAD_SHA1  golang.tar.gz" | sha1sum -c - \
+	&& tar -C /usr/local -xzf golang.tar.gz \
+	&& rm golang.tar.gz
+
+RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
+
+COPY deps/ /tmp
+RUN dpkg -i /tmp/*.deb
```
Comment (cyberdelia) [inline docker/precise/Dockerfile:26]: I don't think there is any deps to install so you can remove theses.

File: .travis.yml (hunk 2 above)
Comment (cyberdelia) [inline .travis.yml:18]: This can be done later in a separate PR.

File: docker/precise/Dockerfile (hunk 5 above)
Comment (gregburek) [inline docker/precise/Dockerfile:26]: is dh_golang not used?

Comment (cyberdelia) [inline docker/precise/Dockerfile:26]: Then you forgot to add the said files to this PR?

Comment (gregburek) [inline docker/precise/Dockerfile:26]: no they are below `docker/precise/deps/dh-golang_1.5~precise1_all.deb`

File: debian/changelog [hunk 6]
```diff
@@ -0,0 +1,5 @@
+pglogplexcollector (0.5) precise; urgency=medium
+
+  * Initial release.
+
+ -- Greg Bureke <gregburek@heroku.com>  Tue, 08 Dec 2015 01:22:12 +0000
```
Comment (gregburek) [inline debian/changelog]: (-_-)

File: docker/precise/Dockerfile (hunk 5 above)
Comment (cyberdelia) [inline docker/precise/Dockerfile:26]: embarassed

**Pull request merged**